jobs/tests/test_utils.py

```diff
@@ -170,15 +170,15 @@
 
     # Thunderbird Mac OS X Release
     {
-        "id": "thunderbird_10-0-12esr_macosx_pt-br",
+        "id": "thunderbird_esr_10-0-12esr_macosx_pt-br",
         "source": {
             "product": "thunderbird",
         },
         "target": {
             "version": "10.0.12esr",
             "platform": "macosx",
             "locale": "pt-BR",
-            "channel": "release"
+            "channel": "esr"
```

Comment:
Oh you are right, that's why we went back and forth about it thanks, I will rework that.